src/class-cache-flusher.php

```diff
@@ -50,4 +56,16 @@ function flush_domain() {
         // Call API and check response code
         return $result->success();
     }
+
+    /**
+     * Flush the OpCache if enabled
+     */
+    private function flush_opcache()
+    {
+        $opcache_status = opcache_get_status();
+
+        if ($opcache_status && is_array($opcache_status) && $opcache_status['opcache_enabled']) {
```

Comment:
opcache_get_status() can return an Array or false. If it is false, the you can't get the index 'opcache_enabled'. Should be:

```
if ($opcache_status && is_array($opcache_status)) {
  if ($opcache_status['opcache_enabled']) {
    // do something
  }
}
```

Reply:
As php checks an and clause from left to right, this if is implicit in the third part of the and.
If $opcache_status == false it fails the and clause and never reaches the array check